mava/systems/jax/ippo/networks.py

```diff
@@ -325,12 +356,11 @@ def make_default_networks(
         net_spec_keys: keys for each agent network
         policy_layer_sizes: policy network layers
         critic_layer_sizes: critic network layers
+        policy_recurrent_layer_sizes: Optionally add recurrent layers to the policy
```

Comment:
Also, please update `tests/jax/components/networks_test.py` to test this functionality/ class. The dummy network factor in that test does not have any recurrent layers

Reply:
I added a test for this.

Reply:
Yeah, it is. Thanks for creating the issue to revisit